src/components/ModelActions/ModelActions.tsx

```diff
@@ -70,7 +70,7 @@ const ModelActions: FC<Props> = ({ modelName, modelUUID }: Props) => {
           },
           {
             children: Label.DESTROY,
-            disabled: isController ?? !canConfigureModel,
+            disabled: isController || !canConfigureModel,
```

Comment:
This change suggests we might need another test, or a test isn't functioning correctly.